src/databricks/sqlalchemy/test_local/e2e/test_basic.py

```diff
@@ -188,6 +188,26 @@ def test_create_table_not_null(db_engine, metadata_obj: MetaData):
     metadata_obj.drop_all(db_engine)
 
 
+def test_column_comment(db_engine, metadata_obj: MetaData):
```

Comment:
When table comments are supported this test can be replaced by the sqlalchemy TCK table comment tests.